Improve forms

tresoldi opened the pull request:
This PR improve the quality of forms in data, especially in terms of parentheses and brackets. Only 1,7% of forms are potentially problematic (down from over 11%), they are now more evenly distributed across languages and parameters, and the false positives in terms of colexifications seem solved.

It does so by using three regexes and by extending both the list of replacements and of hand-corrected lexemes. It would be possible to fix all, but I suppose it is preferable this way, as I am only accounting to what I am relatively sure (also by comparing forms in parentheses with other forms and by manually checking some cognates) and as the presence of errors serves to indicate that the quality of data is not the desirable.

chrzyki reviewed (approved):
Thanks @tresoldi! That looks much better than before. I'll list a couple of things that I've noticed with the cleaned data but I'll let you be the judge whether they're important/worth the time to investigate/worth fixing. Feel free to simply ignore these:

- `uxu-` for Kimbu still appears in a lot of different concepts/forms
- form `enda` is, concept-wise, weirdly heterogeneous
- language Sonjo seems to have a lot of 'footnotes', some of them are still in the forms (e.g. Sonjo-769, form 'baba 1a'); also, the concepts for these values (i.e. those marked with 'footnotes') seem weird, Tuveta (check e.g. Tuveta-105) likewise
- Sonjo-324 `e-` appears to be a cat classifier (might be interesting to check this in Concepticon)
- ProtoBantu has a lot of `\`, `m-`, `a`, `d-`, `i`, `t-`,  forms (due to splitting in the proto forms)
- Nyamwezi has a lot of `+` forms
- Sukuma has `k` as a form a lot for a variety of concepts
- there are a number of `?` forms
- there are a number of `)` forms

tresoldi commented:
I have corrected most of the issues reviewed by @chrzyki , and fixed even more. Problematic entries are now 0,5%.

The `enda` form is heterogeneous indeed, a characteristic maybe deriving from usage as an auxiliary. As it is heterogeneous in the raw data already, I left it as it is. I have also kept the potential classifier of Sonjo, as it would require more study of the language (but @chrzyki hypothesis seems right).

chrzyki commented:
Looks good. Thanks a bunch @tresoldi. :)
